src/traceme.c

```diff
@@ -347,54 +447,128 @@ traceme_start_tracer(struct tracer_conf *tr_conf)
     DEBUG("Travis mode. Not starting tracing");
     return NULL;
 #else
+    int failing = 0;
+    struct tracer_ctx *tr_ctx = NULL;
+    int perf_fd = -1, out_fd = -1;
+    sem_t tracer_init_sem;
```

Comment:
Variable defined before use.

Reply:
OK. FWIW, I still find this style quite jarring for C code.